src/identity/components/Name.tsx

```diff
@@ -41,10 +43,6 @@ export function Name({
     return <span className={className} />;
   }
 
-  if (!name) {
```

Comment:
This PR is over complicating the solution. Keep the solution simple as we spoke over Slack.

```
if (!name) {
  name = getSlicedAddress(accountAddress)
}
```